create-snowpack-app/app-scripts-react/babel.config.json

```diff
@@ -1,9 +1,3 @@
 {
-  "presets": [["@babel/preset-react"], "@babel/preset-typescript"],
-  "plugins": ["@babel/plugin-syntax-import-meta"],
```

Comment:
somewhat related: tested with Melissa to confirm that this is no longer needed (included by default by Babel)